### PR TITLE
IA-1953 attempt to fix tests

### PR DESCRIPTION
**Pull request by Qi77Qi**
1. `6a783a5` doesn't seem to exist in dockerhub..See if this fixes tests..cc @akarukappadath 
2. Use `LeonardoApiClient` in more places for creating runtime
3. Remove deployWelder
4. Attempt to fix NotebookBioconductorKernelSpec based on https://stackoverflow.com/questions/62003082/elementnotinteractableexception-element-not-interactable-element-has-zero-size

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by akarukappadath — APPROVED** [review]
thanks for catching this!

**Comment by Qi77Qi** [comment]
seems like there's something else going on too 🤔  ..Jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoConfig.scala [hunk 1]
```diff
@@ -20,12 +20,14 @@ object LeonardoConfig extends CommonConfig {
...
-    val oldDockerHubWelderDockerImage: String = leonardo.getString("dockerHubWelderUri") + ":" + oldWelderHash
+    val oldGcrWelderDockerImage: ContainerImage =
+      ContainerImage(leonardo.getString("gcrWelderUri") + ":" + oldWelderHash, ContainerRegistry.GCR)
+    val oldDockerHubWelderDockerImage: ContainerImage =
+      ContainerImage(leonardo.getString("dockerHubWelderUri") + ":" + oldWelderHash, ContainerRegistry.DockerHub)
     val curGcrWelderDockerImage: String = leonardo.getString("gcrWelderUri") + ":" + curWelderHash
     val curDockerHubWelderDockerImage: String = leonardo.getString("dockerHubWelderUri") + ":" + curWelderHash
     val bioconductorImageUrl: String = leonardo.getString("bioconductorImageUrl")
-    val rstudioBaseImageUrl: String = leonardo.getString("rstudioBaseImageUrl")
+    val rstudioBaseImageUrl = ContainerImage(leonardo.getString("rstudioBaseImageUrl"), ContainerRegistry.GCR)
```
Comment (akarukappadath) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoConfig.scala]: nit: rename to `rstudioBaseImage`

File: core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala [hunk 2]
```diff
@@ -290,7 +290,19 @@ object JsonCodec {
...
+      se <- c.downField("serverExtensions").as[Option[Map[String, String]]]
+      ce <- c.downField("combinedExtensions").as[Option[Map[String, String]]]
+      le <- c.downField("labExtensions").as[Option[Map[String, String]]]
+    } yield UserJupyterExtensionConfig(ne.getOrElse(Map.empty),
+                                       se.getOrElse(Map.empty),
+                                       ce.getOrElse(Map.empty),
+                                       le.getOrElse(Map.empty))
+  }
+
+  Decoder.forProduct4(
```
Comment (akarukappadath) [inline core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala]: should this be removed?

Comment (Qi77Qi) [reply core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala]: oh good catch! weird this didn't fai compilation

**Review by rtitle — APPROVED** [review]

**Pull request merged**